src/main/java/groovy/util/GroovyCollections.java

```diff
@@ -123,6 +124,15 @@ public static List combinations(Iterable collections) {
         return collectedCombos;
     }
 
+    private static <T> List<T> newArrayList(T... elements) {
+        if (elements == null || elements.length == 0) {
+            return emptyList();
+        }
+        List<T> ret = new ArrayList<>(elements.length);
+        ret.addAll(Arrays.asList(elements));
```

Comment:
Maybe the following code is a bit more efficient ;-)
```java
Arrays.stream(elements).collect(Collectors.toList())
```

Reply:
I agree, 100X times better!